src/ruby/ext/symengine/ruby_symbol.h

```diff
@@ -0,0 +1,9 @@
+#include "ruby.h"
+#include "ruby_basic.h"
+#include "cwrapper.h"
+
+void csymbol_free(void *ptr);
```

Comment:
No, I haven't. But I plan to do that. I was going through a few blog posts trying to find a more Ruby-like way to find memory leaks. Something like rspec. I will give them a try first. Like, [this answer](http://stackoverflow.com/questions/20956401/how-do-i-track-down-a-memory-leak-in-my-ruby-code#20956962) on stackoverflow.